README.md

```diff
@@ -77,7 +77,7 @@ Follow the instructions below or read the <a href="https://github.com/3raxton/sp
 
 ## Support
 
-* If you run into any issues or need help troubleshooting any custom apps or extensions reach out to the Spicetify community on [Spectrum](https://spectrum.chat/spicetify)
+* If you run into any issues or need help troubleshooting any custom apps or extensions reach out to the Spicetify community on [Discord](https://discord.gg/VnevqPp2Rr), or create a [discussion](https://github.com/khanhas/spicetify-cli/discussions) in the spicetify-cli repository
```

Comment:
Hey @Poopooracoocoo thanks for making a Pull Request! I'd like to keep the spectrum link available for users of Spicetify to go back and look at previous issues. If you could leave this in but also add the discord link that would be great.

Sorry for the delay but let's get this merged! 🔥

Reply:
Umm  free to make the changes yourself!! :)

And maybe specify that Spectrum is for "archived issues"